Shared/Confirm-ExchangeShell.ps1

```diff
@@ -102,7 +102,9 @@ function Confirm-ExchangeShell {
 
                     Write-Verbose "Imported Module. Trying Get-Exchange Server Again"
                     try {
-                        Get-ExchangeServer @params | Out-Null
+                        $isEMS = Get-ExchangeServer @params |
+                            Select-Object -First 1 |
+                            ForEach-Object { if ($_.GetType().Name -eq "ExchangeServer") { return $true } return $false }
```

Comment:
Isn't this an expensive way to do this due to the number of Exchange servers? What if we did something like Get-OrganizationConfig instead?